project4/project/ebs/templates/ebs/home.html

```diff
@@ -0,0 +1,809 @@
+				<div class="modal-content">
+					<div class="modal-header modal-color">
+						<button class="pull-right btn2" type="button" data-dismiss="modal" aria-hidden="true">&times;</button>
+						<h2 class="text-center">Password Recovery</h2>
+					</div>
+					<div class="modal-body modal-padding">
+					<form id="forgotform" method='post'>
+						<div id="forgotmessage">  </div>
+						{% csrf_token %}	
+						<br> 
+						<div class="top-margin-20 form-group">
+							<label>Email</label>
+							<input id="email1" type="email"  class="form-control input-lg"  placeholder="email" required>
+						</div>
+
+						<div class="top-margin-30 form-group">
+							<button class="btn btn-block btn-lg btn1">Send My Password</button>
+							<p class="forgot-modal"> Password Reset Link Will Be Email To You</p>
+						</div>   
+				
+					</form>
+					</div>
+				</div>
+		</div>
+</div>
+			
+<!-- recover password Modal -->
+	<div id="recoverformmodal"  class="modal fade" role="dialog">
+		<div class="modal-dialog">
+			<div class="modal-content">
+				<div class="modal-header modal-color">
+					<button class="pull-right btn2" type="button" data-dismiss="modal" aria-hidden="true">&times;</button>
+					<h2>Reset Password</h2>
+				</div>
+				<div class="modal-body modal-padding">
+				<form id="recoverform" data-toggle='validator'>
+				<div id="recovermessage"></div>
+				<br>
+				{% csrf_token %}
+				{% if hashsuccess %}
+					<input type="hidden" id="hashval" value="{{hash}}">
+				{% else %}
+					<input type="hidden" id="hashval" value="">
+				{% endif %}
+				<div class="form-group">
+					<label class="control-label">New Password</label>
+						<div class="col-sm-12 col-xs-12 input-group">
+							<div class="input-group">
+								<span class="input-group-addon transparent"><span class="glyphicon glyphicon-lock">
+								</span>
+								</span>
+								<input id="password1" name="password" type="password" class="form-control"  data-minlength="8" placeholder="New Password" required>
+							</div>
+							<div class="help-block with-errors"></div>
+					
+						</div>
+				</div>
+				<div class="form-group">
+					<label class="control-label">New Password Confirm</label>
+						<div class="col-sm-12 col-xs-12 input-group">
+							<div class="input-group">
+								<span class="input-group-addon transparent"><span class="glyphicon glyphicon-lock">
+								</span>
+								</span>
+								<input id="password2" name="confirmpassword" type="password"
+					  class="form-control" data-minlength="8" placeholder="New Password confirm" required>
+							</div>
+							<div class="help-block with-errors"></div>
+						</div>						
+				</div>
+				<div class="form-group">
+					<button name="submit" class="btn btn-lg btn-block btn1">Change Password</button>
+				</div>   
+				
+				</form>
+				</div>
+			</div>
+		</div>
+	</div>
+
+
+
+
+	<!--signup modal -->
+	<div id="bannerformmodal" class="modal fade" role="dialog">
```

Comment:
Create separate file for very modal, that can be extended in respective file. Need to create new task in next sprint.